src/main/scala/org/apache/mesos/chronos/core/RichRuntime.scala

```diff
@@ -0,0 +1,39 @@
+package org.apache.mesos.chronos.core
+
+import java.util.logging.Logger
+import java.util.{ Timer, TimerTask }
+
+import scala.concurrent.duration._
+import scala.concurrent.{ ExecutionContext, Future, _ }
+
+import akka.Done
+
+class RichRuntime {
+
+  private[this] val logger = Logger.getLogger(getClass.getName)
+
+  def asyncExit(
```

Comment:
I'm not sure about the problem you are solving, but why do you need an `asyncExit()`?